flaggr/src/main/java/com/comuto/flag/model/Flag.java

```diff
@@ -18,6 +18,12 @@
  */
 public class Flag {
 
```

Comment:
A documentation would be welcome here, to explain the purpose of the UNKNOWN state (and why we are moving from a boolean to 3-states).